### DISCUSSION TITLE
Invitación pendiente (expirada) GitHub

### DISCUSSION THREAD

**Discussion by DarlintonPulgarin** [rramosp/ai4eng.v1#314] [00 - General]
Saludos,
Realicé la invitación a los usuarios rramosp y Felipe-RA a mi repositorio el 28 de marzo. Hoy fui a revisar GitHub pues nunca se me notificó en la app móvil que ningún usuario entrara como colaborador y me enteré que nunca aceptaron la invitación; la de Felipe-RA ya no aparece y la de rramosp la indica como expirada. 

![2024-04-25](https://github.com/rramosp/ai4eng.v1/assets/163614765/d8664973-6597-42f8-a556-2533f28d5787)

**Comment by Felipe-RA** [comment]
Hola @DarlintonPulgarin 

Si, justo hoy terminamos de calificar y ya sólo nos quedan aquellos con los que se nos vencieron las invitaciones.

Los estaremos contactando hoy a lo largo del día con instrucciones adicionales.
